### DISCUSSION TITLE
people chat here and tell meh new ideas for hacks

### DISCUSSION THREAD

**Discussion by ghost** [Prodigy-Hacking/ProdigyMathGameHacking#917] [(A) Anything]
hi chat here

**Comment by ghost** [comment]
note all the hacks should be javascript

**Comment by ghost** [comment]
code

**Comment by ghost** [comment]
like this _.player.q13kennel.data.map(x => x.level = 99999999999999999999999999999999999999999999999); It changes ur pet level

**Comment by PatheticMustan** [comment]
locked, please do not impersonate me.
